crowbar_framework/config/routes.rb

```diff
@@ -240,4 +236,48 @@
   # TODO(must): Get rid of this wildcard route
   match "/:controller/:action/*(:.format)",
     via: [:get, :post, :put, :patch, :delete]
+
+  namespace :api,
```

Comment:
@aspiers I think `:api` namespace is totally OK; at least that's what we [do in SCC](https://github.com/SUSE/happy-customer/blob/master/glue/config/routes/api.rb) as well.

If your API mostly follows your UI, reusing the same routing scheme helps. If it's some completely separate beast with a different usage patters (which is the case for SCC's `:connect` and `"import` APIs, and I think for Crowbar APIs too) — it doesn't help that much, just bloats your controllers.

@MaximilianMeister There's a tiny subtlety with the subdomain approach — if you're doing HTTPS on your application (and you probably should), this will require a second certificate. Moreover, in some circumstances that certificate can be unobtainable at all (because it's one level deeper, and some issuers'/clients' policies don't allow that).